employee-rostering-backend/src/test/java/org/optaweb/employeerostering/skill/SkillRestControllerIT.java

```diff
@@ -0,0 +1,164 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.employeerostering.skill;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.optaweb.employeerostering.domain.skill.Skill;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class SkillRestControllerIT {
+
+    @Autowired
+    private TestRestTemplate skillRestTemplate;
+
+    private String skillPathURI = "http://localhost:8080/rest/tenant/{tenantId}/skill/";
+
+    @Test
+    public void getSkillListTest() {
+        Integer tenantId = 1;
+        Integer tenantId2 = 2;
+        String name = "name";
+        String name2 = "name2";
+
+        Skill skill = new Skill(tenantId, name);
+        Skill skill2 = new Skill(tenantId, name2);
+        Skill skill3 = new Skill(tenantId2, name);
+
+        ResponseEntity<Skill> postResponse = skillRestTemplate.postForEntity(skillPathURI + "add", skill, Skill.class
+                , tenantId);
+        ResponseEntity<Skill> postResponse2 = skillRestTemplate.postForEntity(skillPathURI + "add", skill2, Skill.class
+                , tenantId);
+        ResponseEntity<Skill> postResponse3 = skillRestTemplate.postForEntity(skillPathURI + "add", skill3, Skill.class
+                , tenantId2);
+
+        ResponseEntity<List<Skill>> response = skillRestTemplate.exchange(skillPathURI, HttpMethod.GET, null,
+                                                                          new ParameterizedTypeReference
+                                                                                  <List<Skill>>() {}, tenantId);
+        ResponseEntity<List<Skill>> response2 = skillRestTemplate.exchange(skillPathURI, HttpMethod.GET, null,
+                                                                           new ParameterizedTypeReference
+                                                                                   <List<Skill>>() {}, tenantId2);
```

Comment:
Could you please isolate the `skillRestTemplate` calls into reusable method. The tests will become much simpler to understand if we can read something like:
```java
Skill skill = new Skill(tenantId, name);
ResponseEntity<Skill> postResponse = addSkill(skill, tenantId);
ResponseEntity<List<Skill>> response = getSkills(tenantId);
```